core/services/chainlink/testdata/config-full.toml

```diff
@@ -268,6 +268,9 @@ ContractTransmitterTransmitTimeout = '1m0s'
 DatabaseTimeout = '1s'
 ObservationGracePeriod = '1s'
 
+[EVM.Automation]
+TransmitGasLimit = 540
```

Comment:
Why is this a smaller number while others are in the millions? Is there a config multiplier?

Reply:
just dummy data for test